api-reference/50 Common/utils/utils/getTimeZones(date).md

```diff
@@ -5,13 +5,28 @@ export: getTimeZones
 ---
 ---
 ##### shortDescription
-<!-- Description goes here -->
+Gets the list of <a href="https://en.wikipedia.org/wiki/List_of_tz_database_time_zones" target="_blank">IANA</a> time zone objects.
 
 ##### return: Array<dxSchedulerTimeZone>
-<!-- Description goes here -->
+A list of IANA time zone objects.
 
 ##### param(date): Date | undefined
-<!-- Description goes here -->
+A date for which to get time zones. If not passed, the current date is used.
```

Comment:
"A date for which to get time zones" This does not make sense. Time zones are the same regardless of the date.

Reply:
No, they may change due to daylight savings or other reasons. Thus this parameter.

Reply:
Time zones do not change - the offset changes with daylight saving time, but the time zone stays the same. Countries can, on rare occasions, change their time zone. The last country to change its time zone was in 1995 if I remember correctly.